contracts/DixelArt.sol

```diff
@@ -2,11 +2,9 @@
 
 pragma solidity ^0.8.10;
 
-import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
-import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
+import {Ownership, IERC173} from "@beandao/contracts/library/Ownership.sol";
+import {ERC721, IERC721, IERC721Enumerable} from "@beandao/contracts/library/ERC721.sol";
```

Comment:
I'm not sure if it's a good idea using a custom ERC721 contract here instead of OZ's implementation.

I think your implementation could be more efficient and well designed, but I believe OZ's implementation is more secure for following reasons:

1. It's being used by many other projects for a long time, so bugs could be already detected and fixed.
2. We're importing `@beandao/contracts@0.7.0` version here, but I can see that there're some bugs fixed even today

Reply:
That's right. For a long time, Oz kept getting heavy, and the gas was wasted. In fact, Oz's counter is not required at 0.8, but your environment uses more gas at 0.8.
If you use Oz, it's hard to save it more. 😔 but I understand your choice! All of this is a suggestion.

btw `@beandao/contracts@0.7.0` follows semantic versioning.